.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "authorino"]
+	path = authorino
+	url = https://github.com/Kuadrant/authorino/
```

Comment:
Seem a bit overkill that we need to include the entire Authorino repo here. Is this because of the types? I thought we could rely on Envoy protobufs only to integrate the external authorization service.

Reply:
To deploy Authorino, we should rely on https://github.com/kuadrant/authorino-operator now, BTW.

Reply:
It's just there to deploy the e2e example like installing CRDs and deploying authorino itself. Yes, we are using Envoy protobufs present in the `vendor-protobufs`.